doc/encoders.texi

```diff
@@ -3654,6 +3654,14 @@ Set the encoding profile (scc requires libmfx >= 1.32).
 @item scc
 @end table
 
+@item @var{tier}
+Set the encoding tier.
```

Comment:
Could you add description in details ? According to your patch, It only takes effects for level >= 4.

Reply:
According to the HEVC specs, only level >= 4 can support HIGH tier.
I make a check here to avoid wrong settings.